.release-it.beta.json

```diff
@@ -1,9 +1,9 @@
 {
 	"non-interactive": true,
 	"verbose": true,
-	"requireCleanWorkingDir": true,
-	"changelogCommand":
-		"echo \"### Commits\n$(git log --no-merges --pretty=format:'* %s (%h)' [REV_RANGE])\"",
+	"requireCleanWorkingDir": false,
```

Comment:
We need to remember to fix this, I left it in as a mistake. But we should fix it in a real commit, not a merge commit.